.licenserc.yaml

```diff
@@ -1,28 +1,31 @@
 #
-# Copyright 1999-2019 Seata.io Group.
+# Licensed to the Apache Software Foundation (ASF) under one or more
+# contributor license agreements.  See the NOTICE file distributed with
+# this work for additional information regarding copyright ownership.
+# The ASF licenses this file to You under the Apache License, Version 2.0
+# (the "License"); you may not use this file except in compliance with
+# the License.  You may obtain a copy of the License at
 #
-# Licensed under the Apache License, Version 2.0 (the "License");
-# you may not use this file except in compliance with the License.
-# You may obtain a copy of the License at
-#
-# http://www.apache.org/licenses/LICENSE-2.0
+#     http://www.apache.org/licenses/LICENSE-2.0
 #
 # Unless required by applicable law or agreed to in writing, software
 # distributed under the License is distributed on an "AS IS" BASIS,
 # WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 # See the License for the specific language governing permissions and
 # limitations under the License.
 #
+
 header:
   license:
     spdx-id: Apache-2.0
     copyright-owner: Seata.io
     content: |
-      Copyright 1999-2019 Seata.io Group.
-    
-      Licensed under the Apache License, Version 2.0 (the "License");
-      you may not use this file except in compliance with the License.
-      You may obtain a copy of the License at
+      Licensed to the Apache Software Foundation (ASF) under one or more
```

Comment:
copyright-owner: Seata.io  Is that right?

Reply:
thanks for review, I have fixed it.